roles/reproducer/tasks/devscripts_post.yml

```diff
@@ -17,18 +17,24 @@
       ansible.builtin.slurp:
         path: >-
           {{
-            (ansible_user_dir,
```

Comment:
The fact that we need to edit this in a couple of tasks is just suggesting we may want to move the path (till the /auth) to a variable in the block.

Reply:
Thank you. It is a good suggestion. Let me incorporate the change.